### PR TITLE
[Metricbeat] Move statsd metricbeat module to GA

### PR DESCRIPTION
**Pull request by mtojek**
This PR prepares `statsd` module for releasing GA.

What has been done:
* docs update
* tags changed from _beta_ to _ga_
* system test

Issue: https://github.com/elastic/beats/issues/14280

### CODE REVIEW

File: x-pack/metricbeat/module/statsd/_meta/docs.asciidoc [hunk 1]
```diff
@@ -1,2 +1,79 @@
...
+=== Metricsets
+
+Currently, there is only `server` metricset in `statsd` module.
+
+[float]
+==== `server`
+The metricset collects metric data sent using UDP and publishes them under the `statsd` prefix.
+
+[float]
+=== Development
```
Comment (jsoriano) [inline x-pack/metricbeat/module/statsd/_meta/docs.asciidoc]: I think development notes should be in a `README.md` instead.

**Comment by jsoriano** [comment]
Failure in docs build could be related, but not sure.

File: x-pack/metricbeat/module/statsd/_meta/docs.asciidoc (hunk 1 above)
Comment (mtojek) [reply x-pack/metricbeat/module/statsd/_meta/docs.asciidoc]: Fixed.

**Comment by mtojek** [comment]
> Failure in docs build could be related, but not sure.

Yes, I've been working on that in the mean time. Let the CI verify if it's fine now.

**Comment by mtojek** [comment]
> Thanks!

Okey, now it looks fine:

http://beats_16447.docs-preview.app.elstc.co/guide/en/beats/metricbeat/master/_statsd_module.html

**Comment by mtojek** [comment]
> Thanks!

Okey, it need some time to propagate the visualization:

http://beats_16447.docs-preview.app.elstc.co/guide/en/beats/metricbeat/master/metricbeat-module-statsd.html

**Pull request merged**